pkg/k8sutil/k8sutil.go

```diff
@@ -752,6 +761,12 @@ func (k *K8sutil) CreateClientMasterDeployment(deploymentType, baseImage string,
 			},
 		}
 
+		// disable secrets volume mount when ssl is not enabled
```

Comment:
So your recommendation is to remove this code or keep it? Do you have more suggestions for this PR?